### ISSUE TITLE
collect statistics about CA

### ISSUE THREAD

**Issue by evilsocket** [evilsocket/opensnitch#139]
while apps are creating TLS connections, the daemon should (asynchronously) reslove the whole certificates chain of every visited host and collect statistics about what certification authority is being used for each one (incrementing its connection counter in a new `by_ca map[string]int` field. These new stasts should be rendered as a new tab in the UI.

**Comment by akatasonov** [comment]
not bad, haven't seen these stats anywhere else!

**Comment by evilsocket** [comment]
yeah, i'm personally very curious to see from whom my traffic is protected and encrypted ...

**Comment by aes512** [comment]
This would be GREAT! Fantastic idea ;-)
